pkg/model/database_secret.go

```diff
@@ -17,8 +17,8 @@ func DatabaseSecret(cr *v1alpha1.Keycloak) *v1.Secret {
 			},
 		},
 		StringData: map[string]string{
-			DatabaseSecretUsernameProperty: cr.ObjectMeta.Name + "-" + RandStringRunes(4),
-			DatabaseSecretPasswordProperty: cr.ObjectMeta.Name + "-" + RandStringRunes(4),
+			DatabaseSecretUsernameProperty: cr.ObjectMeta.Name + "-" + GenerateRandomString(4),
+			DatabaseSecretPasswordProperty: cr.ObjectMeta.Name + "-" + GenerateRandomString(4),
```

Comment:
Generated password should be longer. In fact since it's only used by Keycloak and doesn't need to be shareable it can be long. 32 characters even.

Reply:
Fixed.